tests/benchmarks/numeric/vanZadelhoff_1_3D_mesher.py

```diff
@@ -54,8 +54,8 @@
 mesh = mesher.Mesh(meshName)
 
 npoints = len(mesh.points)
-# nbs     = [n for sublist in mesh.neighbors for n in sublist]
-# n_nbs   = [len(sublist) for sublist in mesh.neighbors]
+nbs     = [n for sublist in mesh.neighbors for n in sublist]
+n_nbs   = [len(sublist) for sublist in mesh.neighbors]
```

Comment:
Do not use the neighbors derived form the `magritte.mesher.Mesh`.
There are models for which this does not work.
Instead use or adapt the `magritte.setup.set_Delaunay_neighbor_lists` function.
This basically re-Delaunay-tetrahedralizes the given point cloud, but has a better behaved neighbour structure than our derived one.

Reply:
Take this into account in all test scripts.

Reply:
This is now changed for all test scripts.